samples/instance-partition-create.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2024 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// sample-metadata:
+//  title: Creates a new instance partition
+//  usage: node instance-partition-create.js <INSTANCE_ID> <INSTANCE_PARTITION_ID> <PROJECT_ID>
+
+'use strict';
+
+function main(
+  instanceId = 'my-instance',
+  instancePartitionId = 'my-instance-partition',
+  projectId = 'my-project-id'
+) {
+  async function createInstancePartition() {
+    // [START spanner_create_instance_partition]
+    // Imports the Google Cloud client library
+    const {Spanner} = require('@google-cloud/spanner');
+
+    /**
+     * TODO(developer): Uncomment the following lines before running the sample.
+     */
+    // const projectId = 'my-project-id';
+    // const instanceId = 'my-instance';
+    // const instancePartitionId = 'my-instance-partition';
```

Comment:
nit: can we change my-instance-partition to an actual example of a instance partition id. That will make it easier for users to understand how they can use it.
Something like this maybe -  https://github.com/alkatrivedi/nodejs-spanner/blob/main/samples/database-create-with-default-leader.js#L30

Reply:
Actually, "my-instance-partition" is a valid instance partition id. The requirements for instance partition id are very similar to the requirements for instance id.

Reply:
Got it!